scio-test/src/it/scala/com/spotify/ScioIOBenchmark.scala

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2019 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.spotify
+
+import com.google.api.services.bigquery.model.{TableFieldSchema, TableSchema}
+import com.spotify.ScioBenchmarkSettings.{commonArgs, BatchMetrics}
+import com.spotify.scio._
+import com.spotify.scio.coders._
+import org.apache.avro.generic.GenericRecord
+import org.joda.time.DateTimeZone
+import org.joda.time.format.DateTimeFormat
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Future}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.Try
+
+/**
+ * IO benchmark jobs, run once daily and logged to DataStore.
+ *
+ * This file is symlinked to scio-bench/src/main/scala/com/spotify/ScioIOBenchmark.scala so
+ * that it can run with past Scio releases.
+ */
+object ScioIOBenchmark {
+
+  import ScioBatchBenchmark._
+
+  private val benchmarks =
+    BenchmarkCommon.benchmarks("com\\.spotify\\.ScioIOBenchmark\\$[\\w]+\\$")
+
+  private val avroGcsPath =
+    "gs://data-integration-test-benchmark-eu/avro-benchmark/folder-a/folder-b/part-*"
+
+  private val textGcsPath =
+    "gs://data-integration-test-benchmark-eu/text-benchmark/folder-a/part-*"
+
+  def main(args: Array[String]): Unit = {
+    val argz = Args(args)
```

Comment:
the main method logic here is the same as in `ScioBatchBenchmark` right? maybe we could refactor some of the arg parsing into the `Benchark#run()` method?